android/jni/Android.mk

```diff
@@ -5,7 +5,7 @@ $(info TARGET_ARCH_ABI is $(TARGET_ARCH_ABI))
 
 $(info LOCAL_PATH is $(LOCAL_PATH))
 
-PREBUILT_LIBS := ./libs/prebuilt/android-$(TARGET_ARCH_ABI)
+PREBUILT_LIBS := ./libs/eegeo/prebuilt/android-$(TARGET_ARCH_ABI)

```

Comment:
If we didn't do this and don't put platform things inside eegeo folder then by taking the platform update with ./update.platform.sh -p android, it actually clear all libraries and folders from libs. Since we added cardboard.jar from google cardboard to track VR things, it also got removed from libs folder by taking platform update.